src/types/catalog.ts

```diff
@@ -13,7 +13,7 @@ export interface CatalogItem extends Record<string, any> {
   id?: string
   title: string
   description: string
-  key?: string
+  key?: string // optional parameter to be used as key in v-for, has to be unique for each item
```

Comment:
suggestion: If you add as an actual block comment above the property, it will show up in intellisense (same comment for the other instance)

```suggestion
  /** Optional parameter to be used as key in v-for, has to be unique for each item */
  key?: string
```

Reply:
> it will show up in intellisense

You mean like so?
![Screenshot 2024-05-10 at 2 25 17 PM](https://github.com/Kong/kongponents/assets/36751160/cbe188a1-622c-4005-b5c9-3ca4bc213aa3)

Reply:
If that's what you mean than inline comments show up too

Reply:
Or am I getting it wrong?